test/lib/ufe/testMoveCmd.py

```diff
@@ -394,6 +394,7 @@ def createCommonAPI(testCase, sphereXformable):
 
         self.runTestOpUndo(createCommonAPI, 'xformOp:translate')
 
+    @unittest.skipUnless(ufeUtils.ufeFeatureSetVersion() < 3, 'Creating a SceneItem was fixed in UFE v3.14')
```

Comment:
Should test explicitly for UFE 0.3.14, using UFE_PREVIEW_VERSION_NUM, as external builders of maya-usd don't have that UFE version yet.  See e.g. testAttribute.py for details.